kuma/events/models.py

```diff
@@ -96,7 +97,7 @@ def reload(self, data=None):
         header_line = events.pop(0)
         parse_header_line(header_line)
 
-        today = datetime.today()
+        today = timezone.datetime.today()
```

Comment:
Please don't use the timezone module for shadowing the use of the datetime module.